scripts/main.py

```diff
@@ -438,4 +443,4 @@ def start_interaction_loop(self):
 
 
 if __name__ == "__main__":
-    main()
+    main()
```

Comment:
There is whitespace error here. Press Enter once after main(). It won't pass the linter anyway.